src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -834,21 +837,26 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
         };
     }
 
-    private _generateNewHandTipMeshes() {
+    private _generateNewHandTipMeshes(disableNearInteraction: boolean) {

```

Comment:
What is the intended use case for using this flag?

Reply:
Just saw Raanan's comment from a few weeks back, makes sense. Is it possible to set it up in a way that also allows for disabling/enabling near interaction after creation?

Reply:
you mean on every frame? i don't think so